<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="SketchViewer" name="viewer" native="true"/>
      <widget class="QGroupBox" name="tools_groupBox">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QWidget" name="camera_toolBoxPage" native="true">
          <layout class="QFormLayout" name="formLayout_2">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Viewer</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QCheckBox" name="cameramodeCheckBox">
               <property name="text">
                <string>First person view</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="wireframemodeCheckBox">
               <property name="text">
                <string>Wireframe</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="4" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="personheightLabel">
               <property name="text">
                <string>Person height: 2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="personheightSlider">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>2</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="sketching_toolBoxPage" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Sketching</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="resetCameraPushButton">
             <property name="text">
              <string>Change first camera back to last position</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="sketchesTreeWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>90</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>220</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="lineWidth">
              <number>2</number>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="uniformRowHeights">
              <bool>false</bool>
             </property>
             <column>
              <property name="text">
               <string>Sketches</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="sketchmodeCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Sketch mode</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QPushButton" name="addSketchPushButton">
               <property name="text">
                <string>Add Sketch</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="deleteSelectedPushButton">
               <property name="text">
                <string>Delete Selected</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QPushButton" name="deformTerrainPushButton">
                  <property name="text">
                   <string>Deform Terrain</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="lowerSilhouettesPushButton">
                  <property name="text">
                   <string>Lower Silhouettes</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QPushButton" name="detectFeaturesPushButton">
                  <property name="text">
                   <string>Features</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="detectRidgesPushButton">
                  <property name="text">
                   <string>Ridges</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="detectSilhouettesPushButton">
                  <property name="text">
                   <string>Silhouettes</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="matchPushButton">
             <property name="text">
              <string>Match strokes to features</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="embedSketchPushButton">
             <property name="text">
              <string>Embed 2D Silhouettes in 3D</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadTerrain"/>
    <addaction name="actionSaveTerrain"/>
    <addaction name="separator"/>
    <addaction name="actionLoadSketch"/>
    <addaction name="actionSaveSketch"/>
    <addaction name="separator"/>
    <addaction name="actionComputeNoiseStats"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoadTerrain">
   <property name="text">
    <string>Open terrain</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSaveTerrain">
   <property name="text">
    <string>Save Terrain</string>
   </property>
  </action>
  <action name="actionLoadSketch">
   <property name="text">
    <string>Open Sketches</string>
   </property>
  </action>
  <action name="actionSaveSketch">
   <property name="text">
    <string>Save Sketches</string>
   </property>
  </action>
  <action name="actionLoadPickedEdges">
   <property name="text">
    <string>Open Picked Edges</string>
   </property>
  </action>
  <action name="actionComputeNoiseStats">
   <property name="text">
    <string>Print noise stats</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SketchViewer</class>
   <extends>QWidget</extends>
   <header>gui/sketch_viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
